db/schema.rb

```diff
@@ -431,12 +432,12 @@
   end
 
   create_table "devices", force: :cascade do |t|
-    t.bigint "user_id", null: false
-    t.string "token", null: false
-    t.string "platform", null: false
     t.string "app_bundle", null: false
     t.datetime "created_at", precision: 6, null: false
+    t.string "platform", null: false
+    t.string "token", null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.bigint "user_id", null: false
```

Comment:
How do I prevent this change? I went as far as dropping my DB locally, rebuilding from the schema, rebasing with `master` branch, etc. 🤔

Reply:
`git add -p` will ask you about staging each section of a file for commit rather than staging the file wholesale.